code/modules/antagonists/traitor/contractor/datums/contractor_hub_ui.dm

```diff
@@ -15,7 +15,9 @@
 					return
 				page = newpage
 			if("extract")
-				var/error_message = current_contract?.start_extraction_process(ui_host(), usr)
```

Comment:
![image](https://user-images.githubusercontent.com/15887760/101980616-eae2e080-3c66-11eb-9b41-806606d088b9.png)
This compiles. I'd say this is a byond bug that needs byond fixing instead of us using a workaround.
It breaks when you use a `?.` when defining a variable